### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by paylakatel**
This PR:

- makes pop-up content more readable (adds moment library to help with this)
- takes npx out of build/deploy scripts
- fixes issue with crash counts component where it wasn't updating when changing mode
- fixes error handling when calling feature service
- uses moment to get the last updated date for the dataset

### CODE REVIEW

File: components/Map.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import FeatureCounts from '../components/FeatureCounts';
+import moment from 'moment';
```
Comment (fionawhim) [inline components/Map.js]: I'd actually recommend seeing if `date-fns` can do this for you: https://hackernoon.com/why-you-should-choose-date-fns-over-moment-js-in-your-nodejs-applications-116d1a709c43

`moment` has a notoriously large code footprint.

**Review by fionawhim — COMMENTED** [review]
OK to commit after switching to `date-fns` if it seems like it would work.

**Comment by paylakatel** [comment]
date-fns works like a charm! and it's good to know its generally favored over moment.

**Comment by paylakatel** [comment]
how did it get closed?!

**Pull request merged**